network-api/networkapi/templates/404.html

```diff
@@ -1,9 +1,9 @@
-{% extends "pages/base.html" %} {% load wagtailcore_tags wagtailimages_tags %}
+{% extends "pages/base.html" %}
 
 {% block pageTitle %}Page Not Found{% endblock %}
 
+{% block wagtail_metadata %}{% endblock %}
```

Comment:
404 is not managed by wagtail in the slightest, so it shouldn't try to inject wagtail metadata.

Reply:
So shouldn't we do the same thing in mozfest/404?

Reply:
that's a really good point - made sure that's the case